content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,645 @@
+
+`GET workspaces/:workspace_id/varsets`
+
+| Parameter            | Description      |
+| -------------------- | -----------------|
+| `:workspace_id`      | The workspace ID |
+
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "varset-mio9UUFyFMjU33S4",
+      "type": "varsets",
+      "attributes":  {
+         "name": "varset-b7af6a77",
+         "description": "Full of vars and such for mass reuse",
+         "is-global": false,
+         "updated-at": "2021-10-29T17:15:56.722Z",
+         "var-count":  5,
+         "workspace-count": 2
+      },
+      "relationships": {
+        "organization": {
+          "data: {"id": "organization_1", "type": "organizations"}
+        },
+        "vars": {
+          "data": [
+           {"id": "var-abcd12345", "type": "vars"},
+           {"id": "var-abcd12346", "type": "vars"},
+           {"id": "var-abcd12347", "type": "vars"},
+           {"id": "var-abcd12348", "type": "vars"},
+           {"id": "var-abcd12349", "type": "vars"}
+          ]
+        },
+        "workspaces": {
+          "data": [
+           {"id": "ws-abcd12345", "type": "workspaces"},
+           {"id": "ws-abcd12346", "type": "workspaces"}
+          ]
+        }
+      }
+    }
+  ],
+  "links": {
+    "self": "app.terrafor.io/app/my_organization/varsets",
+    "first": "app.terrafor.io/app/my_organization/varsets?page=1",
+    "prev": null,
+    "next": null,
+    "last": "app.terrafor.io/app/my_organization/varsets?page=1"
+  }
+}
+```
+
+### Variable Relationships
+
+## Add Variable
+
+`POST varsets/:varset_external_id/relationships/vars`
+
+| Parameter            | Description                      |
+| -------------------- | ---------------------------------|
+| `:varset_id`         | The variable set ID              |
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+
+Key path                                 | Type   | Default | Description
+-----------------------------------------|--------|---------|------------
+`data.type`                              | string |         | Must be `"vars"`.
+`data.attributes.key`                    | string |         | The name of the variable.
+`data.attributes.value`                  | string | `""`    | The value of the variable.
+`data.attributes.description`            | string |         | The description of the variable.
+`data.attributes.category`               | string |         | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.
+`data.attributes.hcl`                    | bool   | `false` | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.
+`data.attributes.sensitive`              | bool   | `false` | Whether the value is sensitive. If true then the variable is written once and not visible thereafter.
```

Comment:
Docs say:

"Terraform runs receive the full text of [sensitive](#sensitive-values) Terraform variables and may print the value in logs and state files if the configuration sends the value to an output or a resource parameter. Sentinel mocks downloaded from runs will also contain the sensitive values of Terraform variables."

Maybe we shuld change this to read "If true, variable is not visible in the UI."